ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -303,6 +303,10 @@ tests:
   steps:
     cluster_profile: openstack
     workflow: openshift-e2e-openstack-ipi
+- as: openstack-manifests
+  steps:
+    cluster_profile: openstack
```

Comment:
This means the job is going to consume 1 lease from the `openstack` boskos lease pool, but AFAIK this isn't consuming any resources from the underlying openstack cloud. Is there a way to avoid it?

Reply:
Good catch! Let's see if removing the Boskos args (and secret) is enough for ci-operator

Reply:
Automation refuses to let me just remove Boskos. I'll have to create a new cluster_profile; can we merge as-it-is and keep a note?